packages/orchestration/src/examples/unbond.flows.js

```diff
@@ -30,11 +30,7 @@ export const unbondAndTransfer = async (orch, { zcf }) => {
   const stride = await orch.getChain('stride');
   const strideAccount = await stride.makeAccount();
 
-  // TODO the `TIA` string actually needs to be the Brand from AgoricNames
-  // const tiaAmt = await celestiaAccount.getBalance('TIA');
-  // await celestiaAccount.transfer(tiaAmt, strideAccount.getAddress());
-  // TODO https://github.com/Agoric/agoric-sdk/issues/10017
-  // await strideAccount.liquidStake(tiaAmt);
-  console.log(osmoAccount, strideAccount);
+  const balance = await osmoAccount.getBalance('OSMO');
```

Comment:
`OSMO` works, and it doesn't need to be `uosmo`? Maybe grab `stakingTokens[0]` from chainInfo?